pkg/innerring/contracts.go

```diff
@@ -67,23 +76,38 @@ func parseContracts(cfg *viper.Viper, morph *client.Client, withoutMainNet, with
 	}
 
 	for _, t := range targets {
-		*t.dest, err = parseContract(cfg, morph, t.cfgName, t.nnsName)
+		*t.dest, err = parseContract(nnsCtx, _logger, cfg, morph, t.cfgName, t.nnsName)
 		if err != nil {
 			name := strings.TrimPrefix(t.cfgName, "contracts.")
 			return nil, fmt.Errorf("can't get %s script hash: %w", name, err)
 		}
 	}
 
-	result.alphabet, err = parseAlphabetContracts(cfg, morph)
+	result.alphabet, err = parseAlphabetContracts(nnsCtx, _logger, cfg, morph)
 	if err != nil {
 		return nil, err
 	}
 
 	return result, nil
 }
 
-func parseAlphabetContracts(cfg *viper.Viper, morph *client.Client) (alphabetContracts, error) {
-	num := GlagoliticLetter(cfg.GetUint("contracts.alphabet.amount"))
+func parseAlphabetContracts(ctx *nnsContext, _logger *logger.Logger, cfg *viper.Viper, morph *client.Client) (alphabetContracts, error) {
+	var num GlagoliticLetter
+	const numConfigKey = "contracts.alphabet.amount"
```

Comment:
Just out of curiosity, has anyone ever used this? I see it comes from #405, but I doubt it should exist, especially in the merged IR/CN scenario.

Reply:
right now I don't see any real use and general practical applicability of this